src/course-page/CoursePage.jsx

```diff
@@ -15,8 +15,8 @@ function CoursePageConfigTile({
 }) {
   const getContainerClasses = () => {
     return classNames('bg-white shadow p-3 m-3', {
-        'border-disabled': !coursePageStatus,
-        'border-enabled': coursePageStatus,
+        'course-page-title.disabled': !coursePageStatus,
+        'course-page-title': coursePageStatus,
     });
```

Comment:
This can be simplified to:
```javascript
    return classNames('bg-white shadow p-3 m-3 course-page-title', {disabled: !coursePageStatus});
```

Reply:
That didn't work as intended because course-page-title.disabled is required, so I had to use a variation.

Reply:
@taniwha that might be because you are defining the SCSS styling incorrectly. 
Note: 
```scss
.class1 {
 color: red;
 .class2 {
   color: green;
 }
}
```
In the above the inner class2 will not match a div with both  class1 and class2, It will match a child element of class2 inside an element with class1. 

If you want to match an element that has both class1 and class2 (in this case both `course-page-title` and `disabled`), the correct scss code is:
```scss
.class1 {
 color: red;
 &.class2 {
   color: green;
 }
}
```